src/TorchSharp/Autograd.cs

```diff
@@ -33,66 +33,13 @@ public void Dispose(bool disposing)
             }
         }
 
-        public static bool IsAutogradEnabled()
-        {
-            return THSAutograd_isGradEnabled();
-        }
+        public static bool IsAutogradEnabled() => THSAutograd_isGradEnabled();
     }
 
     public static partial class torch
     {
-        /// <summary>
-        /// Context-manager that disables gradient calculation.
-        /// </summary>
-        /// <returns></returns>
-        public static IDisposable no_grad() => new AutoGradMode(false);
-
-        /// <summary>
-        /// Context-manager that enables gradient calculation.
-        /// </summary>
-        /// <returns></returns>
-        public static IDisposable enable_grad() => new AutoGradMode(true);
-
-        /// <summary>
-        /// Context-manager that sets gradient calculation to on or off.
-        /// </summary>
-        /// <returns></returns>
-        public static IDisposable set_grad_enabled(bool mode) => new AutoGradMode(mode);
-
-        /// <summary>
-        /// Returns true if grad mode is currently enabled.
-        /// </summary>
-        /// <returns></returns>
-        public static bool is_grad_enabled() => AutoGradMode.IsAutogradEnabled();
-
-        public static partial class autograd
+        public static class autograd
```

Comment:
indeed. a user of the library might want to add custom methods in this namespace. however, it would be cleaner if the user would put custom methods in a dedicated class.